pkg/proxy/replica.go

```diff
@@ -35,7 +36,30 @@ func (p *Proxy) ReplicaList(ctx context.Context, req *rpc.ProxyEngineRequest) (r
 	}
 	ackResponse(resp.ProxyEngineResponse, false)
 
-	// TODO
+	c, err := eclient.NewControllerClient(req.Address)
```

Comment:
As I mentioned before, can we store/cache the client in this proxy server? The idea should be similar to `ProxyHandler` in the manager PR. If YES, you can create a ticket & add a TODO.

Reply:
https://github.com/longhorn/longhorn/issues/3984